examples/ola-streaming-client.cpp

```diff
@@ -41,6 +41,12 @@ DEFINE_s_string(dmx, d, "", "Comma separated DMX values to send, e.g. "
 DEFINE_s_uint32(universe, u, 1, "The universe to send data for");
 DEFINE_uint8(priority, ola::dmx::SOURCE_PRIORITY_DEFAULT,
              "The source priority to send data at");
+DEFINE_s_default_bool(universe_from_source, s, false,
+                      "Read universe number from source. Only allowed when "
+                      "processing STDIN. The universe number must be formatted "
+                      "in base 10, and be separated by at least one whitespace"
```

Comment:
What's the actual format on STDIN?
```
Uni chan1,chan2
Uni chan1,chan2
```

Is this also valid?:
```
Uni chan1,chan2 Uni chan1,chan2
```

Just a bit thrown by your end comment.

Reply:
Yeah, any sort of whitespace serving as a delimiter should be fine.

It was using https://en.cppreference.com/w/cpp/string/basic_string/operator_ltltgtgt to tokenize the input, and that (by default) just extracts chunks of input delimited by whitespace.

Reply:
Added some examples in the CLI help: 6ff5ae190daa8260e985f3a6996ecaf2f10ddc75